packages/components/organisms/f-registration/src/components/Registration.vue

```diff
@@ -190,6 +189,15 @@ import EventNames from '../event-names';
  */
 const meetsCharacterValidationRules = value => /^[\u0060\u00C0-\u00F6\u00F8-\u017Fa-zA-Z-' ]*$/.test(value);
 
+/**
+ * Tests that the user is not retrying a conflicted email address
+ *
+ * @param {string} value The email address to test.
+ * * @param {object} vm The Vue instance
+ * @return {boolean} True if the email does not email the conflicted email address value
+ */
+const emailNotConflicted = (value, vm) => (value !== vm.conflictedEmailAddress);
```

Comment:
Since this returns a boolean, could you please rename the function so it begins with "has", "can", "should", etc?